GUI/Controls/AllModVersions.cs

```diff
@@ -139,8 +139,18 @@ public GUIMod SelectedModule
                 }
                 catch (ModuleNotFoundKraken)
                 {
-                    // No versions to be shown, abort and hope an auto refresh happens
-                    return;
+                    // Identifier unknown to registry, maybe installed from local .ckan
+                    // Take the module associated with GUIMod, if any
+                    var module = value.ToModule();
+                    if (module == null)
+                    {
+                        VersionsListView.Items.Clear();
+                        return;
+                    }
+                    allAvailableVersions = new Dictionary<CkanModule, bool>()
+                    {
+                        { module, installable(installer, module, registry) }
+                    };
```

Comment:
It seems like the installed module should also be included in the list even if other versions of the mod are found (for example, if I wanted to test `PlanningNode-v0.1.3.ckan` before it is released). Can we move this logic outside the `catch` block to do that?

Reply:
Doing that,good suggestion.
What should we do if the dict already contains a module with this version?
If we replace the existing module, you won't see updated compat data in case it changes in the repository.
If we don't replace it, the tab would show the compat data of the repository even if you installed it from .ckan and it differs. You might also come to the conclusion that the install somehow didn't work because the compat data didn't update.

I don't see a way to reliably differentiate between outdated data due to repo changes and "newer" data from local .ckan.
I'd opt for not replacing to display the repo data since this is what the average user might encounter and benefit from. Installing from local .ckans is something probably only we do, and we can deal with that.

Reply:
> What should we do if the dict already contains a module with this version?

I think I'm fine with whatever you choose there. This is a pretty rare case to begin with. One option not considered so far is to list both if we find that they're different, so for example you might see that there is a 1.5.0 version that is compatible with 1.11 but the 1.5.0 that you have installed is only compatible with 1.10. That could be pretty confusing though.

Reply:
> One option not considered so far is to list both if we find that they're different

I did try that already, but a dict can only have unique keys, and `CkanModule`s are considered equal if the `identifier` and `version` matches...

I'm going with my preferred option then, so users at least see the "correct" version range in case it changed since they installed it.